src/index.js

```diff
@@ -742,7 +742,10 @@ export function useDropzone(options = {}) {
         return;
       }
 
-      if (event.keyCode === 32 || event.keyCode === 13) {
+      if (
```

Comment:
We'll obviously need some unit tests. There's already a bunch of them testing the `keyCode`.

Reply:
I'll look into it when I have the time!